src/Backoffice/Pagination/PaginationRequest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Pagination;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class PaginationRequest extends FormRequest
+{
+    public const PAGE = 'page';
+
+    public const PER_PAGE = 'per_page';
+
+    public function rules(): array
+    {
+        return [
+            self::PAGE => [
+                'integer',
+                'min:1',
+                'sometimes',
+            ],
+            self::PER_PAGE => [
+                'integer',
+                'min:1',
+                'max:100',
+                'sometimes',
```

Comment:
nit pick, i like putting the `required`/`sometimes` as the first validation so you know easier if they are required or not, but you can leave it like this